public/locales/en/translation.json

```
@@ -0,0 +1,28 @@
+{
+    "home": {
+        "title": "Federal centers of software development",
+        "welcome": "Welcome to the federal centers of software development",
+        "wsgCard": {
+            "title": "WorldSkills Germany",
+            "body": "Visit the website WorldSkills Germany"
+        },
+        "ytCard": {
+            "title": "YouTube",
+            "body": "To our YouTube-channel"
+        },
+        "igCard": {
+            "title": "Instagram",
+            "body": "To our Instagram-profile"
+        },
+        "fbCard": {
+            "title": "Facebook",
+            "body": "To our Facebook-profile"
+        }

```

In English, these compound nouns are not hyphenated

```suggestion
        "ytCard": {
            "title": "YouTube",
            "body": "To our YouTube channel"
        },
        "igCard": {
            "title": "Instagram",
            "body": "To our Instagram profile"
        },
        "fbCard": {
            "title": "Facebook",
            "body": "To our Facebook profile"
        }

```